### PR TITLE
PixelPaint: Fixed some Color Masking issues

### PR DESCRIPTION
**Pull request by Torstennator**
This change fixes an issue where pixelpaint was crashing when trying to refine an editing mask with the "Color Masking" feature because we where accessing pixel coordinates that where beyond the boundaries. Furthermore a change was made to make the color masking hue selection work again with mouse clicks directly on the color wheel.

### CODE REVIEW

**Comment by nico** [comment]
https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy

> Write the commit message subject line in the imperative mood ("Foo: Change the way dates work", not "Foo: Changed the way dates work").

And since it's two independent things, maybe it should be two commits?

Otherwise looks good, thanks for the PR.

File: Userland/Applications/PixelPaint/ImageMasking.cpp [hunk 1]
```diff
@@ -437,8 +437,8 @@ double ColorWheelWidget::hue()
 void ColorWheelWidget::calc_hue(Gfx::IntPoint const& position)
 {
     auto center = Gfx::IntPoint(width() / 2, height() / 2);
+    auto angle = AK::to_degrees(AK::atan2(static_cast<float>(position.y() - center.y()), static_cast<float>(position.x() - center.x())));
```
Comment (nico) [inline Userland/Applications/PixelPaint/ImageMasking.cpp:440]: The commit message ends mid-sentence :)

Comment (Torstennator) [reply Userland/Applications/PixelPaint/ImageMasking.cpp:440]: Sorry for that... it has already been late after work.

**Comment by nico** [comment]
Thanks!

**Pull request merged**